Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -379,39 +393,42 @@ N/A
 During development of a mobile application, traditional techniques for IPC might be applied like usage of shared files or network sockets. As mobile application platforms implement their own system functionality for IPC, these mechanisms should be applied as they are much more mature than traditional techniques. Using IPC mechanisms with no security in mind may cause the application to leak or expose sensitive data.
 
 The following is a list of Android IPC Mechanisms that may expose sensitive data:
-* Binders<sup>[1]</sup>
-* Services<sup>[2]</sup>
-  * Bound Services<sup>[9]</sup>
-  * AIDL<sup>[10]</sup>
-* Intents<sup>[3]</sup>
-* Content Providers<sup>[4]</sup>
+
+-  Binders<sup>[1]</sup>
+-  Services<sup>[2]</sup>
+-  Bound Services<sup>[9]</sup>
+-  AIDL<sup>[10]</sup>
```

Comment:
It seems "Bound Services" and "AIDL" were indented in the old version, and not in the new version. Is this intentional?